subsys/tfm/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+set_property(GLOBAL PROPERTY
```

Comment:
This looks quite cryptic on its own, could you add a comment explaining why this is needen?

Also, could you double check that we don't have any ZEPHYR_BINARY_DIR or similar aliases for CMAKE_BINARY_DIR that we should use instead? I always mix these up and don't remember when we should use what, but just double check what the idiomatic way is.

Reply:
I don't think we do, for this case. @tejlmand can comment if I'm wrong.